third_party/stdlib/dummy_thread.py

```diff
@@ -0,0 +1,145 @@
+"""Drop-in replacement for the thread module.
```

Comment:
Lock is similar to Mutex in Go. In the short term I think something like this would work:

```
class Lock(object):
  def __init__(self):
    self._mutex = Mutex.new()

  def acquire(self):
    self._mutex.Lock()

  def release(self):
    self._mutex.Unlock()

  def __enter__(self):
    self.acquire()

  def __exit__(self, *args):
    self.release()
```

Long term there are some behavior differences to work around since acquire() can be non-blocking, but Go mutex does not support that mode.